tests/test_coils.py

```diff
@@ -204,13 +204,27 @@ def test_adding_MagneticField_with_Coil_or_CoilSet(self):
     def test_converting_coil_types(self):
         """Test conversions between coil representations."""
         s = np.linspace(0, 2 * np.pi, 100, endpoint=False)
-        coil1 = FourierRZCoil(1e6, [0, 10, 2], [-2, 0, 0])
+        coil1 = FourierRZCoil(1e6, [0, 10, 0], [0, 0, 0])
```

Comment:
The `FourierPlanarCoil` fails to fit the `FourierRZCoil` well if I make the n=1 component of it anything but zero... this seems like something it should be able to do with sufficient resolution? Maybe I messed something up in my latest commit on the fit?